core/src/test/java/com/google/googlejavaformat/java/MainTest.java

```diff
@@ -119,6 +119,37 @@ public void testMain() throws Exception {
     assertThat(process.exitValue()).isEqualTo(0);
   }
 
+  @Test
+  // https://stackoverflow.com/questions/7660651/passing-command-line-unicode-argument-to-java-code
+  public void testMainWithUnicodeArgumentFailsOnWindows() throws Exception {
```

Comment:
Did you want to include this test, or was it more to document the problem? I guess it would provide some notice if this changed?

Reply:
It only demonstrates the underlying issue why I needed that `--sourceFile` feature.

I think, it's superseded by now and I'll remove it from this PR.